pages/apis/graphql/graphql_cookbook.md

```diff
@@ -82,6 +82,48 @@ query RecentPipelineSlugs {
 }
 ```
 
+### Get a pipeline's UUID
```

Comment:
I think it would be good to frame this heading more around the task rather than the answer. It looks like the question came from someone wanting a way to refer to pipelines that won't change over time. Perhaps:

```suggestion
### Get a pipeline's persistent identifier
```

What do you think? I feel like there might be a better term. I like "permanent" more than persistent but it looks like [persistent identifiers](https://en.wikipedia.org/wiki/Persistent_identifier) are a thing. I also thought of "fixed" but don't want to cause any confusion with fixing a failing pipeline.

Reply:
Hmm, I do really like the idea. I think the problem is that it is a UUID (universal unique identifier) which denotes it's uniqueness as an identifier.

What if instead, we used the wording "Get a pipeline's persistent UUID"

Reply:
👍🏻 I'm good with that

Reply:
Updated with new commit! :D